worker/draft.go

```diff
@@ -997,10 +997,13 @@ func (n *node) updateRaftProgress() error {
 
 func (n *node) checkpointAndClose(done chan struct{}) {
 	slowTicker := time.NewTicker(time.Minute)
+	lastSnapshotTime := time.Now()
 	defer slowTicker.Stop()
 
-	snapshotAfter := x.WorkerConfig.Raft.GetUint64("snapshot-after")
-	x.AssertTruef(snapshotAfter > 10, "raft.snapshot-after must be a number greater than 10")
+	snapshotAfterEntries := x.WorkerConfig.Raft.GetUint64("snapshot-after-entries")
+	x.AssertTruef(snapshotAfterEntries > 10, "raft.snapshot-after must be a number greater than 10")
+
+	snapshotAfterDuration := x.WorkerConfig.Raft.GetDuration("snapshot-after-duration")
```

Comment:
names look fine, but longer names are generally avoided in local functions. Please see if it can be named something shorter that makes sense.